modules/internal/String.chpl

```diff
@@ -482,11 +482,11 @@ module String {
     inline proc size return len;
 
     /*
-      :returns: The number of Unicode codepoints in the string.
+      :returns: The number of codepoints in the string (typically Unicode).
```

Comment:
> (typically Unicode)

Is it possible to elaborate on this in the docs? E.g., when would the string codepoints be treated not as unicode?

Reply:
Heh . . . Unicode vs. other multibyte encodings is controversial.  We said as much as we could without imposing undesired restrictions in #12385 .  I think I will just delete the reference to Unicode here.

Reply:
Sounds good.